sdk/boms/spring-cloud-azure-dependencies/pom.xml

```diff
@@ -50,89 +50,27 @@
       <!-- Azure SDK -->
       <dependency>
         <groupId>com.azure</groupId>
-        <artifactId>azure-core</artifactId>
-        <version>1.26.0</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
-      </dependency>
-      <dependency>
-        <groupId>com.azure</groupId>
-        <artifactId>azure-cosmos</artifactId>
+        <artifactId>azure-cosmos</artifactId> <!-- Spring Cloud Azure is tested with 4.28.0 -->
         <version>4.28.0</version> <!-- {x-version-update;com.azure:azure-cosmos;dependency} -->
```

Comment:
@pallavit please take a look at this, cosmos releases two versions this month, [CHANGELOG.md#4280-2022-03-18](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/cosmos/azure-cosmos/CHANGELOG.md#4280-2022-03-18), but the azure-sdk-bom contains the first version, but the version used in [our code is 4.28.0](https://github.com/Azure/azure-sdk-for-java/blob/a38962a7549d6a837b46b7ad379eed3a4603acd2/sdk/spring/spring-cloud-azure-starter-cosmos/pom.xml#L93-L97). So here instead of using the version from the SDK bom, I chose the 4.28.0 in our bom.